subsys/usb/class/hid/core.c

```diff
@@ -7,7 +7,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#define LOG_LEVEL CONFIG_USB_DEVICE_LOG_LEVEL
+#define LOG_LEVEL CONFIG_USB_DEVICE_HID_LOG_LEVEL
 #include <logging/log.h>
 LOG_MODULE_REGISTER(usb_hid);
```

Comment:
if LOG_LEVEL is not used in the code self, then it can be simplified to
`LOG_MODULE_REGISTER(usb_hid, CONFIG_USB_DEVICE_HID_LOG_LEVEL);`

Reply:
Yea, I know but I wanted to stay consistent with other classes in this matter. [cdc_acm](https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/usb/class/cdc_acm.c#L57), [dfu](https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/usb/class/usb_dfu.c#L55), [bluetooth](https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/usb/class/bluetooth.c#L22).

I know there there is still come inconsistency, eg. cdc_acm Kconfig option is:
USB_CDC_ACM_LOG_LEVEL_<lvl>
vs: hid
USB_**DEVICE**_ HID_LOG_LEVEL_<lvl>

What would You suggest here? IMO USB_DEVICE_<class_name>_LOG_LEVEL_<lvl> is better and I could adapt other classes as well.

Reply:
I prefer USB_ HID_LOG_LEVEL, shorter and there is no need to differ between host or device.

Reply:
done.